.gitignore

```diff
@@ -15,3 +15,33 @@ vendor/
 
 # vim temp files
 *.swp 
+
+# Created by https://www.gitignore.io/api/vim
```

Comment:
damn vim, you noisy lol

Reply:
Yeah haha, I don't think it's all necessary, but I like using gitignore.io to generate them just in case